src/lib/ng-v5/packagr.ts

```diff
@@ -117,8 +117,9 @@ export class NgPackagr {
       buildTransformOperator,
       catchError(err => {
         // Report error and re-throw to subscribers
-        log.error(err);
-        throw err;
+        const msg = process.env.DEBUG || !(err instanceof Error) ? err : err.message;
+        log.error(msg);
+        throw throwError(err);
```

Comment:
Was this meant to throw the `Observable` returned from `throwError`? I would expect `return` here.

Reply:
Opps good catch